### PR TITLE
fix(sidenav): fix the sidnav at small screen sizes

### PR DESCRIPTION
**Pull request by davidicus**
**Summary**

The sidenav gets collapsed at small screen sizes and does reveal when toggled. This PR fixes this/

**Acceptance Test (how to verify the PR)**

- Open up preview link. Go to sidenav. Shrink screen till it gets to small  screen size. You should see the icons disappear when it gets low enough. Now click on hamburger menu. Sidenav should now appear.

### CODE REVIEW

File: src/components/SideNav/SideNav.jsx [hunk 1]
```diff
@@ -14,155 +14,6 @@ import { COLORS, PADDING } from '../../styles/styles';
...
-//       top: 0;
-//       bottom: 0;
-//       left: 0;
-//       width: 4px;
-//       background-color: ${COLORS.blue60};
-//     }
-//   }
-// `;
-
-// new styles
```
Comment (tay1orjones) [inline src/components/SideNav/SideNav.jsx]: Can we delete all this commented-out styling?

Comment (davidicus) [reply src/components/SideNav/SideNav.jsx]: This is deleted. the pink is showing the deletion of this.

Comment (tay1orjones) [reply src/components/SideNav/SideNav.jsx]: 🤦‍♂ clearly I am out of it, this _is_ deleted

**Pull request merged**